openhcl/underhill_attestation/src/lib.rs

```diff
@@ -383,16 +401,16 @@ pub async fn initialize_platform_security(
 
     tracing::info!(
         CVM_ALLOWED,
-        state_refresh_request_from_gsp,
-        vm_id_changed,
+        state_refresh_request_from_gsp = state_refresh_request_from_gsp,
+        vm_id_changed = vm_id_changed,
```

Comment:
I believe this (and other similar ones) can be simplified to just `vm_id_changed,`. no need to assign to the same identifier

Reply:
Just following the conventions of the file to have specify key=value pair.s

Reply:
Knowing when something is an intentional convention vs. an unintentional convention isn't easy. I can't say for certain whether this is indeed totally intentional or not, but I would lean towards this being an unintentional convention.

I suggest fixing up any new instances of this redundant k=v assignment, and as part of this broader cleanup you're doing, also fix up the existing instances as well

Reply:
Done.